perses/tests/test_topology_proposal.py

```diff
@@ -336,6 +337,71 @@ def test_mutate_from_alanine():
         else:
             _ = generate_dipeptide_top_pos_sys(ala.topology, amino, ala.system, ala.positions, system_generator, conduct_geometry_prop=False)
 
+def test_protein_atom_maps():
+    
```

Comment:
it looks like you hardcoded some specific mutations. is this robust enough? i guess you can augment this test as/if you encounter problems in future?

Reply:
I think this is robust enough, as the test cases are representative of the different cases handled by the code: 
1) mutations where the old or new residue is GLY
2) mutations where the old or new residue is in ['THR', 'ILE', 'VAL']
3) all other mutations